src/test/java/BogdanSandu/tests/Tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,53 @@
+package BogdanSandu.tests.Tema6;
+
+import BogdanSandu.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.Set;
+
+public class DemoQaWindow {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) {
+        navigateToDemoQAWindowPage();
+        clickOnNewTabButton();
+        closeBrowser();
+    }
+
+    public static void navigateToDemoQAWindowPage() {
+        driver = BrowserManager.createDriverAndGetPage();
+        driver.get("https://demoqa.com/browser-windows");
+        System.out.println("Am deschis Demo QA window page!");
+    }
+
```

Comment:
Add interactions with other buttons too - there are two more on this page, not only the one showed at retro. Thanks!